packages/shorebird_cli/test/src/commands/init_command_test.dart

```diff
@@ -247,6 +255,106 @@ If you want to reinitialize Shorebird, please run "shorebird init --force".''',
       expect(exitCode, ExitCode.software.code);
     });
 
+    test('throws software error when unable to detect schemes (non-macos)',
+        () async {
+      when(() => platform.isMacOS).thenReturn(false);
```

Comment:
This is stylistic and completely optional, but I've found that this kind of test becomes slightly more readable when put in groups:

```
group('on macOS', () {
  setUp(() {
    when(() => platform.isMacOS).thenReturn(true);
  });

  test('throws software error when unable to detect schemes', ...)
});

group('on a non-mac OS', () {
  setUp(() {
    when(() => platform.isMacOS).thenReturn(false);
  });

  test('throws software error when unable to detect schemes', ...)
});
```

Reply:
yup good call will reorganize.